cfscrape/__init__.py

```diff
@@ -152,20 +153,26 @@ def solve_cf_challenge(self, resp, **original_kwargs):
         body = resp.text
         parsed_url = urlparse(resp.url)
         domain = parsed_url.netloc
-        submit_url = "%s://%s/cdn-cgi/l/chk_jschl" % (parsed_url.scheme, domain)
+        if self.org_method is None:
+            self.org_method = resp.request.method
+        submit_url = "%s://%s/%s" % (parsed_url.scheme,
+                                     domain,
+                                     re.findall('\<form id=\"challenge-form\" action=\"\/(.*)\?', resp.text)[0])
 
         cloudflare_kwargs = copy.deepcopy(original_kwargs)
 
         headers = cloudflare_kwargs.setdefault("headers", {})
         headers["Referer"] = resp.url
 
         try:
-            params = cloudflare_kwargs["params"] = OrderedDict(
+            cloudflare_kwargs["data"] = OrderedDict(
                 re.findall(r'name="(s|jschl_vc|pass)"(?: [^<>]*)? value="(.+?)"', body)
             )
+            params = cloudflare_kwargs["params"] = {'__cf_chl_jschl_tk__': re.findall(
+                '\<form id=\"challenge-form\" action=\".*\?__cf_chl_jschl_tk__=(.*?)\"', resp.text)[0]}
```

Comment:
```suggestion
                '\<form id=\"challenge-form\" action=\".*(?:\?:\&)__cf_chl_jschl_tk__=(.*?)\"', resp.text)[0]}
```
On a request to https://kissanime.ru/Anime/Meiken-Lassie/Episode-024?id=164477 the form's acction attrib is 
`action="/Anime/Meiken-Lassie/Episode-024?id=164477gg=aa&__cf_chl_jschl_tk__...` , the first pattern wont work since there is a `&` before `__cf_chl_jschl_tk__`

Reply:
i forgot about other params
its better if we dd hem as kwargs rather than text in url so that there will be no error when requesting
as some rare casing requests can rasie error when adding params to url that has paramsinto it
so i will use 
`\<form id=\"challenge-form\" action=\".*\?(.*?)\"`
to get full param items then will split hem with '&'
and lastly will split them with '=' and add them as kwargs to params